src/v7/import-admin-theme-types.ts

```diff
@@ -0,0 +1,21 @@
+import { readFile, writeFile } from "fs/promises";
+
+import { formatCode } from "../util/format-code.util";
+
+/**
+ * Imports types from `@comet/admin-theme` in vendors.d.ts to allow using custom theme variants and colors
+ */
+export default async function importAdminThemeTypes() {
+    const filePath = "admin/src/vendors.d.ts";
```

Comment:
Should we add a check if whether the file exists here?

Reply:
Done 7ae0dc4ee45ce6bde1a83c6bf6ea6c9c81606cee